src/hssm/plotting/model_cartoon.py

```diff
@@ -0,0 +1,1985 @@
+        Line width for trajectory paths
+    alpha_trajectories : float, default=0.5
+        Transparency of trajectory paths
+    color_trajectories : str or list or dict, default="black"
+        Color(s) for trajectories. Can be:
+        - str: Single color for all trajectories
+        - list: List of colors mapped to possible choices
+        - dict: Mapping of choices to colors
+    **kwargs
+        Additional keyword arguments passed to plotting functions
+
+    Notes
+    -----
+    This function visualizes multiple simulated decision paths, optionally highlighting
+    the response times and choices. Each trajectory shows the evidence accumulation
+    process leading to a decision.
+    """
+    color_dict = {
+        -1: "black",
+        0: "black",
+        1: "green",
+        2: "blue",
+        3: "red",
+        4: "orange",
+        5: "purple",
+        6: "brown",
+    }
+
+    # Check trajectory color type
+    if isinstance(color_trajectories, str):
+        color_trajectories_dict = {}
+        for value_ in sample[0]["metadata"]["possible_choices"]:
+            color_trajectories_dict[value_] = color_trajectories
+    elif isinstance(color_trajectories, list):
+        cnt = 0
+        for value_ in sample[0]["metadata"]["possible_choices"]:
+            color_trajectories_dict[value_] = color_trajectories[cnt]
+            cnt += 1
+    elif isinstance(color_trajectories, dict):
+        color_trajectories_dict = color_trajectories
+    else:
+        pass
+
+    # Make bounds
+    b = np.maximum(sample[0]["metadata"]["boundary"], 0)
+    b_init = b[0]
+    n_roll = int((sample[0]["metadata"]["t"][0] / delta_t_graph) + 1)
+    b = np.roll(b, n_roll)
+    b[:n_roll] = b_init
+
+    # Trajectories
+    for i in range(n_trajectories):
+        tmp_traj = sample[i]["metadata"]["trajectory"]
+        tmp_traj_choice = float(sample[i]["choices"].flatten())
+
+        for j in range(len(sample[i]["metadata"]["possible_choices"])):
+            tmp_maxid = np.minimum(
+                np.argmax(np.where(tmp_traj[:, j] > -999)), t_s.shape[0]
+            )
+
+            # Identify boundary value at timepoint of crossing
+            b_tmp = b[tmp_maxid + n_roll]
+
+            axis.plot(
+                t_s[:tmp_maxid] + sample[i]["metadata"]["t"][0],
+                tmp_traj[:tmp_maxid, j],
+                color=color_dict[j],
+                alpha=alpha_trajectories,
+                linewidth=linewidth_trajectories,
+                zorder=2000 + i,
+            )
+
+            if highlight_rt_choice and tmp_traj_choice == j:
+                axis.scatter(
+                    t_s[tmp_maxid] + sample[i]["metadata"]["t"][0],
+                    b_tmp,
+                    marker_size_rt_choice,
+                    color=color_dict[tmp_traj_choice],
+                    alpha=1,
+                    marker=marker_type_rt_choice,
+                    zorder=2000 + i,
+                )
```

Comment:
Consider defining some names and reusing them. For example, `metadata = sample[i]["metadata"]`.